debian/upstream/metadata

```diff
@@ -0,0 +1,7 @@
+---
+Archive: GitHub
+Bug-Database: https://github.com/erminas/node-red-contrib-revpi-nodes/issues
+Contact: https://github.com/erminas/node-red-contrib-revpi-nodes/issues
+Name: node-red-contrib-revpi-nodes
+Repository: https://github.com/erminas/node-red-contrib-revpi-nodes.git
+Repository-Browse: https://github.com/erminas/node-red-contrib-revpi-nodes
```

Comment:
Is upstream (e.g. erminas) the correct contact for issues and should the links in `Repository` and `Repository-Browse` lead to them, not to us? If I'm understanding correctly the repository served is this one (RevolutionPi/node-red-...), not the one from erminas.

Reply:
No, this is correct as it is the upstream metadata not the package metadata.